app/models/computacenter_user_change.rb

```diff
@@ -0,0 +1,3 @@
+class ComputacenterUserChange < ApplicationRecord
```

Comment:
Where we have a `Computacenter` prefix, could this actually be in the `Computacenter::` namespace (& corresponding sub-directory) instead?